pallets/subtensor/src/benchmarks.rs

```diff
@@ -424,8 +426,13 @@ reveal_weights {
       weight_values.clone(),
       salt.clone(),
       version_key,
-  ));
-    let _ = Subtensor::<T>::commit_weights(<T as frame_system::Config>::RuntimeOrigin::from(RawOrigin::Signed(hotkey.clone())), netuid, commit_hash);
+    ));
+    let _ = Subtensor::<T>::commit_weights(
+        <T as frame_system::Config>::RuntimeOrigin::from(RawOrigin::Signed(hotkey.clone())),
+        netuid,
+        commit_hash,
+        nonce
+    );
```

Comment:
In benchmark better to not silently fail
```suggestion
    Subtensor::<T>::commit_weights(
        <T as frame_system::Config>::RuntimeOrigin::from(RawOrigin::Signed(hotkey.clone())),
        netuid,
        commit_hash,
        nonce
    ).unwrap();
```